hudi-client/hudi-client-common/src/main/java/org/apache/hudi/index/HoodieIndex.java

```diff
@@ -80,7 +81,7 @@ public O updateLocation(O writeStatuses, HoodieEngineContext context,
   @PublicAPIMethod(maturity = ApiMaturityLevel.EVOLVING)
   public abstract <R> HoodieData<HoodieRecord<R>> tagLocation(
       HoodieData<HoodieRecord<R>> records, HoodieEngineContext context,
-      HoodieTable hoodieTable) throws HoodieIndexException;
+      HoodieTable hoodieTable, Option<String> instantTime) throws HoodieIndexException;
```

Comment:
this is a public api. we might have to deprecate and add a new one if we wish to change the signature

Reply:
the api is marked as "Evolving" so changes are expected in major release